### PR TITLE
create and style see blog page for donation app

### LINKED ISSUE #9: Create and style see blog for DonationApp

### PR DESCRIPTION
**Pull request by omaraburabie3**
relates #9

### CODE REVIEW

File: See blog/see-blog.css [hunk 1]
```diff
@@ -1,6 +1,210 @@
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 150%;
+  width: 305px;
+  height: 50px;
+  left: 16px;
+  top: 94px;
+  margin-left: 16px;
+  margin-top: 20px;
```
Comment (emanjaber) [inline See blog/see-blog.css]: There is too much space between the `header` and the `top-par` section, so you can delete `margin-top`

File: See blog/see-blog.html [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
     
-</body>
-</html>
\ No newline at end of file
+    <section class="top-par">
+      <p>The Inspiring People Helping With Coronavirus Efforts</p>
+    </section>
+    
+    <section class="author">
+      <div class="person"><img src="../images\Ellipse 36 (1).png" /></div>
```
Comment (emanjaber) [inline See blog/see-blog.html:31]: The `img` is too small, you need to make it bigger, especially in the desktop size

File: See blog/see-blog.css [hunk 3]
```diff
@@ -1,6 +1,210 @@
...
+    text-align: center;
+  }
+  .author {
+    margin-left: 100px;
+    margin-top: 30px;
+  }
+  .photo {
+    margin-left: 100px;
+    width: 600px;
+    height: 500px;
```
Comment (emanjaber) [inline See blog/see-blog.css]: You need to fix the size of this photo in tablet size, I think `height: 300px` will be suitable

**Review by emanjaber — CHANGES_REQUESTED** [review]
Great job, Omar!
These are some changes that I hope you will make

File: See blog/see-blog.css [hunk 4]
```diff
@@ -1,6 +1,210 @@
...
+  justify-content: space-between;
+  align-items: center;
+  padding: 2px;
+  height: 50px;
+}
+.left-side {
+  width: 24px;
+  height: 24px;
+  left: 40px;
+  top: 74px;
```
Comment (Bas-Shiekh) [inline See blog/see-blog.css]: delete left and top  because they don't change anything

File: See blog/see-blog.css [hunk 5]
```diff
@@ -1,6 +1,210 @@
...
+    margin-left: 180px;
+    margin-top: 20px;
+    width: 3000px;
+    height: 100px;
+  }
+  .photo {
+    margin-left: 150px;
+    width: 1000px;
+    height: 500px;
+    background: url(../images/Rectangle\ 1485.png);
```
Comment (Bas-Shiekh) [inline See blog/see-blog.css]: don't repeat **background** in media query because that make many issues in this site

**Review by Bas-Shiekh — APPROVED** [review]

**Review by Bakeza — APPROVED** [review]

**Review by SaraDahman — APPROVED** [review]

**Review by emanjaber — APPROVED** [review]
**Well done, Omar! 🎉**

**Pull request merged**